### PR TITLE
add serial testgrid for netd

### PR DESCRIPTION
**Pull request by cadmuxe**

### CODE REVIEW

File: config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml [hunk 1]
```diff
@@ -1069,3 +1069,64 @@ periodics:
...
+      - --env=KUBE_MASTER_EXTRA_METADATA=user-data=/go/src/github.com/containerd/cri/test/e2e/master.yaml,containerd-configure-sh=/go/src/github.com/containerd/cri/cluster/gce/configure.sh,containerd-env=/workspace/test-infra/jobs/e2e_node/containerd/containerd-master/env
+      - --env=KUBE_NODE_EXTRA_METADATA=user-data=/go/src/github.com/containerd/cri/test/e2e/node.yaml,containerd-configure-sh=/go/src/github.com/containerd/cri/cluster/gce/configure.sh,containerd-env=/workspace/test-infra/jobs/e2e_node/containerd/containerd-master/env
+      - --extract=gs://koonwah-repo-pub/devel/v1.11.2-beta.0.20+21777efd99bd59
+      - --gcp-node-image=gci
+      - --gcp-nodes=4
+      - --gcp-zone=asia-southeast1-a
+      - --ginkgo-parallel=1
+      - --provider=gce
+      - --test_args=--ginkgo.focus=[Disruptive\]||\[Feature:PerformanceDNS\] --minStartupPods=8
+      - --timeout=120m
```
Comment (fejta) [inline config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml]: This appears to have two, wildly different timeout values?

File: config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml [hunk 2]
```diff
@@ -1069,3 +1069,64 @@ periodics:
       - --test_args=--ginkgo.focus=\[sig-network\] --ginkgo.skip=\[Slow\]|\[Flaky\]|\[Feature:Networking-IPv6\]|\[Disruptive\]|\[Feature:ServiceLoadBalancer\]|\[Feature:PerformanceDNS\] --minStartupPods=8
       - --timeout=120m
       image: gcr.io/k8s-testimages/kubekins-e2e:v20180730-8b7ab3104-master
+
+- interval: 2h
+  name: ci-kubernetes-e2e-containerd-gce-calico-serial
```
Comment (fejta) [inline config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml]: It isn't obvious to me what this is testing, but it doesn't seem like the kubernetes/kubernetes repo given that this is using a fixed version of kubernetes (`v1.11.2-beta.0.20+21777efd99bd59`)

File: config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml [hunk 3]
```diff
@@ -1069,3 +1069,64 @@ periodics:
...
+      - "--timeout=340"
+      - --root=/go/src
+      - --repo=github.com/containerd/cri=master
+      - "--repo=k8s.io/examples=master"
+      - --scenario=kubernetes_e2e
+      - --
+      - --check-leaked-resources
+      - --env=KUBE_MASTER_EXTRA_METADATA=user-data=/go/src/github.com/containerd/cri/test/e2e/master.yaml,containerd-configure-sh=/go/src/github.com/containerd/cri/cluster/gce/configure.sh,containerd-env=/workspace/test-infra/jobs/e2e_node/containerd/containerd-master/env
+      - --env=KUBE_NODE_EXTRA_METADATA=user-data=/go/src/github.com/containerd/cri/test/e2e/node.yaml,containerd-configure-sh=/go/src/github.com/containerd/cri/cluster/gce/configure.sh,containerd-env=/workspace/test-infra/jobs/e2e_node/containerd/containerd-master/env
+      - --extract=gs://koonwah-repo-pub/devel/v1.11.2-beta.0.20+21777efd99bd59
```
Comment (fejta) [inline config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml]: Is this master + patches? Instead could this check out your repo and build kubernetes?

File: config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml (hunk 1 above)
Comment (cadmuxe) [reply config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml]: thank you for point it out, I have updated the timeout value.

File: config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml (hunk 3 above)
Comment (cadmuxe) [reply config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml]: yes, I will change back to the official repo after PR and cherry pick. 
could we leave it for now? 
One pr got merged https://github.com/kubernetes/kubernetes/pull/66685
one is under reviewing https://github.com/kubernetes/kubernetes/pull/67105

File: config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml (hunk 2 above)
Comment (cadmuxe) [reply config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml]: it's k8s 1.11 with containerd, calico and netd enabled.

**Pull request merged**